aspnetcore/signalr/java-client.md

```diff
@@ -0,0 +1,68 @@
+---
+title: ASP.NET Core SignalR Java client
+author: mikaelm12
+description: Learn how to use the ASP.NET Core SignalR Java client.
+monikerRange: '>= aspnetcore-2.2'
+ms.author: mimengis
+ms.custom: mvc
+ms.date: 08/30/2018
+uid: signalr/java-client
+---
+# ASP.NET Core SignalR Java Client
+
+By [Mikael Mengistu](https://twitter.com/MikaelM_12)
+
+The Java client enables connecting to an ASP.NET Core SignalR server from Java code, including Android apps. Like the [JavaScript client](xref:signalr/javascript-client) and the [.NET client](xref:signalr/javascript-client), the Java client enables you to receive and send and receive messages to a hub in real time. The Java client is available in ASP.NET Core 2.2 or later.
```

Comment:
Fix .NET client link

Reply:
"receive and send and receive"

Reply:
"available in ASP.NET Core 2.2 ~~or~~ **and** later"
I think "and" sounds better?